src/components/Basket.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import cx from 'classnames'
+
+// import BasketItems from './BasketItems'
+class Basket extends React.Component {
+  constructor(props){
+    super(props)
+    this.state ={
+      finalPrice: '',
+      showOrder: false
+      
+    }
+
+    this.handleSubmit = this.handleSubmit.bind(this)
+    this.fetchOrder = this.fetchOrder.bind(this)
+  }
+
+  componentDidMount(){
+    const { order } = this.props;
+    let prices = Array.from(order).map(items => items.price);
+    let totalPrice = prices.reduce(function(acc, item) {
+      return acc + item;
+    }, 0);
+    let priceWithDelivery = totalPrice + 5.00
+    this.setState({
+      finalPrice: priceWithDelivery
+    });
+  }
+
+  fetchOrder(){
+    fetch('/order', {
+    method: 'post',
+    body: JSON.stringify(this.props.order),
+    headers: {
+      'Content-Type': 'application/json'
+    }})
+  .then(response => response.json())
+  .then(data => {
+    alert('Order Confirmation Sent To Email')
+    const showOrder = this.props.order
+    this.setState({
+      showOrder: showOrder,
+      button: showOrder ? true : false
```

Comment:
I think this line is not necessary as it duplicates value stored in `showOrder`. You can use `this.state.showOrder` to determine whether to display show order button in render method